.github/workflows/ci.yml

```diff
@@ -16,42 +19,29 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-#      - name: Cache node modules
-#        uses: actions/cache@v2
-#        if: ${{ !env.ACT }}
-#        env:
-#          cache-name: cache-node-modules
-#        with:
-#          # npm cache files are stored in '~/.npm' on Linux/macOS
-#          path: ~/.npm
-#          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-#          restore-keys: |
-#            ${{ runner.os }}-build-${{ env.cache-name }}-
-#            ${{ runner.os }}-build-
-#            ${{ runner.os }}-
-
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
 
-      - name: Install dependencies
-        run: |
-          npm i -g bower grunt-cli
-          npm ci
+      - name: Install Grunt and Bower
+        run: npm i -g bower grunt-cli --force
+
+      - name: Install npm dependencies
+        run: npm ci
 
       - name: Copy the missing _sprite_share.scss file - related to GCWeb#1737 about wet-boew#cc340a6 commit
         run: |
           curl https://gist.githubusercontent.com/duboisp/d69787b300eb1f4d40f937508e10d013/raw/2c25ce9ff228394e626404a35c62f7bc047b13e7/_sprites_share.scss >> _sprites_share.scss
           mv _sprites_share.scss node_modules/wet-boew/src/plugins/share/sprites/_sprites_share.scss
 
-      - name: Checkout wet-boew latest build
+      - uses: actions/checkout@v4
         if: ${{ env.ACT }}
```

Comment:
Because of the condition it doesn't look like this is currently being run. Maybe better to just remove the change or this step